LibreNMS/Alert/Transport/Victorops.php

```diff
@@ -41,7 +41,7 @@ public function contactVictorops($obj, $opts)
         $url = $opts['url'];
 
         $protocol = array(
-            'entity_id' => ($obj['id'] ? $obj['id'] : $obj['uid']),
+            'entity_id' => str($obj['id'] ? $obj['id'] : $obj['uid']),
```

Comment:
should be `(string)` to cast to string.

Reply:
```suggestion
            'entity_id' => strval($obj['id'] ? $obj['id'] : $obj['uid']),
```

Reply:
Sorry was so excited that I fixed it I forgot the function that PHP used to convert a variable to a string.

Reply:
sure

Reply:
Make sure you test it with the correct code :)

Reply:
Just confirmed that is the correct code I tested with, sorry about the mix up.